README.md

```diff
@@ -1,42 +1,69 @@
-# What's inside
+# Containerized Ansible: What's inside #
 
-This container provide ansible toolstack inside a container.
+This container provides the ansible toolstack inside a container.
 * '''Dockerfile''' with the definition of the ansible container
-** based on OpenSUSE Tumbleweed
-** installs ansible and some additional tools
-* System Setup
-** Make sure needed packages installed on the host
----
-# sudo transactional-update pkg install podman python310-rpm
----
-*** Install ansible commands wrapers
-If current user is root /usr/local/bin is updated, for non-root users
-the user's ~/bin area is updated.
----
-as root:
-# (cd ~; podman container runlabel root-install ansible)
-as non-root
-# (cd ~; podman container runlabel user-install ansible)
----
-*** Ansible Commands
----
-ansible
-ansible-community
-ansible-config
-ansible-connection
-ansible-console
-ansible-doc
-ansible-galaxy
-ansible-inventory
-ansible-playbook
-ansible-pull
-ansible-vault
----
-
-*** Uninstall ansible commands wrapers
----
-as root:
-# (cd ~; podman container runlabel root-unstall ansible)
-as non-root
-# (cd ~; podman container runlabel user-unstall ansible)
----
+  * based on OpenSUSE Tumbleweed
+  * installs ansible and some additional tools
+## System Setup ##
+* Make sure needed packages installed on the host
+  * sudo transactional-update pkg install podman python310-rpm
+  * system reboot is required after all transactional updates
+    * sudo shutdown -r now
+
+## Ansible commands
+The ansible commands are provided as symlinks to ansible-wrapper.sh.
+The commands will instanciate the container and execute the ansible
+command.
+
+## To install ansible commands ##
+
+* as root:
+  * for the root user the ansible commands are placed in /usr/local/bin
+  * (cd /; podman container runlabel install ansible)
+* as non-root
+  * for non-root users the ansible commands are placed in the user's ~/bin
```

Comment:
Since we require the user to cd to ~ before running the podman command, this line probably should look something like:

```suggestion
  * for non-root users the ansible commands should placed in the user's ~/bin, so changes to the users home directory first.
```

Reply:
"should placed" is not quite right. If the example is followed, then the commands will be placed in the user ~/bin area. 

I could expand the section to read:
For non-root users  'podman container runlabel user-install ansible' will place the ansible commands in ${PWD}/bin. The following will install the ansible commands into the current users bin area (~/bin).